CMakeLists.txt

```diff
@@ -98,6 +98,15 @@ option(SSG_SCIENTIFIC_LINUX_DERIVATIVES_ENABLED "If enabled, Scientific Linux de
 
 option(FORCE_VALIDATE_EVERYTHING "If enabled, perform all validation tests regardless of oscap version requirements. By default, the build system may not perform validation of some content types if the openscap used to build it is not up-to-date" FALSE)
 
+# Remove this option when we would like to run ansible-lint and yamllint against our playbooks by default.
+# Right now these checks are not performed and need to be enabled by adding -DANSIBLE_CHECKS=ON to cmake
+# before running ctest.
+option(ANSIBLE_CHECKS "Set to ON to enable ansible-lint and yamllint checks" OFF)
+if(ANSIBLE_CHECKS)
+# ANSIBLE_CHECKS needs to have individual playbooks per rule enabled
```

Comment:
Maybe add a warning so if anyone sets `-DSSG_ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED=OFF` explicitly on the CLI, they know why it was enabled? Though default is FALSE. :-) 

Optionally, you could move this even higher up and have the default value of `SSG_ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED` differ based on `ANSIBLE_CHECKS` value. That'll correctly show the warning only when a user explicitly calls `-DANSIBLE_CHECKS=ON -DSSG_ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED=OFF` and catch the conflict. :-) 

Just a thought.

Reply:
I have added a more elaborated condition and a warning in case it's set to OFF

Reply:
Looks good, thanks @ggbecker!